packages/text-field/src/vaadin-lit-text-field.js

```diff
@@ -3,7 +3,7 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/input-container/src/vaadin-input-container.js';
+import '@vaadin/input-container/src/vaadin-lit-input-container.js';
```

Comment:
Tried this on the dev page and it required me to import 
```js
import '@vaadin/input-container/theme/lumo/vaadin-input-container-styles.js';
import '@vaadin/text-field/theme/lumo/vaadin-text-field-styles.js';
import '@vaadin/text-field/src/vaadin-lit-text-field.js';
```

There shouldn't be a need to import `vaadin-input-container-styles.js`, so [this](https://github.com/vaadin/web-components/blob/b3e14000cddc3b5623d6836eab5d929aa8511d63/packages/text-field/theme/lumo/vaadin-text-field.js#L6) should be moved [here](https://github.com/vaadin/web-components/blob/main/packages/text-field/theme/lumo/vaadin-text-field-styles.js)

Reply:
Thanks, created #6579.